internal/sharedcomponent/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+metadata:
+  yaml:
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
This

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L118C1-L119C1
```
internal/sharedcomponent/                                @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
```